orderedmap.go

```diff
@@ -136,3 +136,17 @@ func (m *OrderedMap) Back() *Element {
 		Value:   element.value,
 	}
 }
+
+// Copy returns a new OrderedMap with the same elements.
+// The elements are deep copied. That is changes to the original OrderedMap will
+// not affect elements in the new OrderedMap returned.
+// Using Copy while there are concurrent writes may mangle the result.
```

Comment:
It is not a deep copy because the elements themselves are not recursively copied. How about this:

```
// Copy returns a new OrderedMap with the same elements.
// Using Copy while there are concurrent writes may mangle the result.
```